opennlp-tools/src/main/java/opennlp/tools/entitylinker/LinkedSpan.java

```diff
@@ -16,11 +16,11 @@
 
 package opennlp.tools.entitylinker;
 
+import opennlp.tools.util.Span;

```

Comment:
If we reorder imports with commits we should do it all the same way. 

1. We should update the code conventions page
2. Make sure this is done identical by Eclipse and Intellij through sharing config files